devicon.json

```
@@ -14966,4 +14685,25 @@
         "color": "#443e38",
         "aliases": []
     }
+    {
+        "name": "gamemaker",
+        "altnames": [],
+        "tags": [
+            "gml",
+            "engine",
+            "game-engine"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "line",
+            ],
+            "font": [
+                "plain",
+            ]
+        },
+        "color": "#ffffff",
+        "aliases": []
```

```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            }
        ]
```
add an alias for  plain, since there is no `plain.svg` file